lib/builder/items.rb

```diff
@@ -1,34 +1,40 @@
 # frozen_string_literal: true
 
-module ItemBuilder
+module Builder
   class Item
-    attr_accessor :items,
+    attr_accessor :item_details,
                   :id,
                   :name,
                   :price,
                   :quantity
 
     def build
-      copy_from_individual_fields unless items_exist?
-      items.map! do |part|
-        item = ::Item.new
-        item.id = part[:id]
-        item.name = part[:name]
-        item.price = part[:price]
-        item.quantity = part[:quantity]
-        item
-      end
-      items
+      copy_from_individual_fields unless item_exists?
+      item = ::Item.new
+      item.id = item_details[:id]
+      item.name = item_details[:name]
+      item.price = item_details[:price]
+      item.quantity = item_details[:quantity]
+      item
+    end
+
+    def add_item(id:, name:, price:, quantity: )
```

Comment:
I originally thought you'd build an array of items in here - maybe `add_item` is a poor name